routes/api.php

```diff
@@ -53,10 +53,12 @@
 
 // api to fetch all todo tasks
 Route::get('task', [\App\Http\Controllers\TaskController::class, 'index']);
-
 // comment post request
 
 
+Route::get('archived-tasks', [\App\Http\Controllers\TaskController::class, 'getArchivedTasks']);
```

Comment:
A route is enough for the API you are to create. Having multiple routes for an API to just fetch archived tasks is, at best, redundant.